### DISCUSSION TITLE
Firefox enhanced cookie clearing

### DISCUSSION THREAD

**Discussion by gingerlime** [Cookie-AutoDelete/Cookie-AutoDelete#1208] [General] [answered]
Just saw that Firefox [introduced enhanced cookie clearing](https://blog.mozilla.org/security/2021/08/10/firefox-91-introduces-enhanced-cookie-clearing/).

(How) is this going to affect Cookie-AutoDelete ?

**Comment by kennethtran93** [comment, marked as answer]
As the new feature requires strict mode, all cookie webextensions, including CAD, would continue to not properly work under strict mode. For now, continue using standard mode or change the cookie preference under custom to cross-site tracking cookies.

Bug report to reference is currently here: https://bugzilla.mozilla.org/show_bug.cgi?id=1669716

**Reply by kennethtran93** [reply]
See number #807 for updates as to when this will be properly implemented.
